examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -52,17 +55,19 @@ CHIP_ERROR Commands::RunCommand(ChipDeviceController & dc, NodeId remoteId, int
 
     VerifyOrExit(argc > 1, err = CHIP_ERROR_INVALID_ARGUMENT);
 
-    for (size_t i = 0; i < commands.size(); i++)
+    for (auto & cluster : clusters)
```

Comment:
This is going to enumerate "clusters" in a random order in general, right?  Is that ok for our purposes?  I'd think we want something stable here...

Reply:
I thought it was going to be in the order of the key (which is a string).

Reply:
Ah, ok.  In that case, great.